### DISCUSSION TITLE
Pasted Osiris eucheats.com!!!

### DISCUSSION THREAD

**Discussion by plisskien** [danielkrupinski/Osiris#3481] [General]
So TabzWare very braindead yt made vid about eucheats, and ngl check this out... Be careful and don't buy this shit...
![New Project (3)](https://user-images.githubusercontent.com/29129602/147585191-254c8ea4-dde6-46c6-9cd1-7f030bb96ac7.png)
TabzWare eucheats vid: https://www.youtube.com/watch?v=-p_2uTvWIeE

**Comment by Frouk3** [comment]
This is just pasted imgui
